nvflare/lighter/impl/he.py

```diff
@@ -28,6 +31,11 @@ def __init__(
         scheme="CKKS",
     ):
 
+        if ts is None:
+            print(
```

Comment:
should we using logging instead of print ? 

should the message indicate how to include teaseal? something like 
"To install dependencies with tenseal: pip install -r requirements-extra.txt"